examples/string_filter/src/main.rs

```diff
@@ -0,0 +1,14 @@
+use polars::lazy::prelude::*;
+use polars::prelude::*;
+
+fn main() {
```

Comment:
Can this return `PolarsResult<()>`. Then we don't have to `unwrap`.

Reply:
Yes, I pushed a commit with the suggested change.